src/templates/layout.nunjucks

```diff
@@ -1,24 +1,25 @@
 <!DOCTYPE HTML>
 <html lang="en">
+
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="copyright" content="Copyright 2016 all rights reserved">
     <meta name="authors" content="Jan Werkhoven, Richard Verheyen and Hannah Sutton">
     <meta name="robots" content="index, follow">
-    <meta name="google-site-verification" content="ioMjIIMP2lu2mdNGjdrooe7XUsQ2AIbCtzigPclkQmE"/>
-    {% block meta %}{% endblock %}
+    <meta name="google-site-verification" content="ioMjIIMP2lu2mdNGjdrooe7XUsQ2AIbCtzigPclkQmE" /> {% block meta %}{% endblock %}
```

Comment:
This fucks me up. Don't do this please. It's your autoformatter in Atom that made this one line. Please revert.